doc/varnish/vcl/varnish4.vcl

```diff
@@ -113,7 +113,7 @@ sub ez_purge {
         }
 
         if (req.http.X-Location-Id) {
-            ban("obj.http.X-Location-Id ~ " + req.http.X-Location-Id);
+            ban("obj.http.X-Location-Id ~ \b" + req.http.X-Location-Id) + "\b";
```

Comment:
looks like a typo, the `+ "\b"` should be inside the parenthesis of the ban function.